### PR TITLE
chore(localizations): Improve pt-BR translations

### PR DESCRIPTION
**Pull request by GustavoOS**

### CODE REVIEW

File: packages/localizations/src/pt-BR.ts [hunk 1]
```diff
@@ -914,16 +914,16 @@ export const ptBR: LocalizationResource = {
...
+      formButton: 'Entre na lista de espera',
+      subtitle: 'Digite seu e-mail e te avisaremos quando seu lugar estiver disponível.',
+      title: 'Entre na lista de espera',
     },
     success: {
-      message: undefined,
-      subtitle: undefined,
-      title: undefined,
+      message: 'Você será redirecionado em breve...',
+      subtitle: 'Entraremos em contato quando seu lugar estiver disponível.',
```
Comment (LauraBeatris) [inline packages/localizations/src/pt-BR.ts:925]:
```suggestion
      subtitle: 'Entraremos em contato quando seu lugar estiver disponível',
```

File: packages/localizations/src/pt-BR.ts [hunk 2]
```diff
@@ -525,18 +525,18 @@ export const ptBR: LocalizationResource = {
...
-      actionText: undefined,
-      blockButton__emailSupport: undefined,
-      blockButton__joinWaitlist: undefined,
-      subtitle: undefined,
-      subtitleWaitlist: undefined,
-      title: undefined,
+      actionLink: 'Entre',
+      actionText: 'Já possui um cadastro?',
+      blockButton__emailSupport: 'Suporte por e-mail',
+      blockButton__joinWaitlist: 'Entre na lista de espera.',
```
Comment (LauraBeatris) [inline packages/localizations/src/pt-BR.ts:531]:
```suggestion
      blockButton__joinWaitlist: 'Entre na lista de espera',
```

File: packages/localizations/src/pt-BR.ts [hunk 3]
```diff
@@ -16,57 +16,57 @@ export const ptBR: LocalizationResource = {
...
+      resendButton: 'Não recebeu um código? Reenviar',
+      subtitle: 'Para continuar, insira o código enviado para o seu telefone.',
+      title: 'Verifique seu telefone',
     },
     totpMfa: {
-      formTitle: undefined,
-      subtitle: undefined,
-      title: undefined,
+      formTitle: 'Código de verificação',
+      subtitle: 'Para continuar, insira o código gerado pelo seu aplicativo de autenticação.',
```
Comment (LauraBeatris) [inline packages/localizations/src/pt-BR.ts:68]:
```suggestion
      subtitle: 'Para continuar, insira o código gerado pelo seu aplicativo de autenticação',
```

File: packages/localizations/src/pt-BR.ts [hunk 4]
```diff
@@ -16,57 +16,57 @@ export const ptBR: LocalizationResource = {
...
+      title: 'Verifique seu telefone',
     },
     phoneCodeMfa: {
-      formTitle: undefined,
-      resendButton: undefined,
-      subtitle: undefined,
-      title: undefined,
+      formTitle: 'Código de verificação',
+      resendButton: 'Não recebeu um código? Reenviar',
+      subtitle: 'Para continuar, insira o código enviado para o seu telefone.',
```
Comment (LauraBeatris) [inline packages/localizations/src/pt-BR.ts:63]:
```suggestion
      subtitle: 'Para continuar, insira o código enviado para o seu telefone',
```

File: packages/localizations/src/pt-BR.ts [hunk 5]
```diff
@@ -16,57 +16,57 @@ export const ptBR: LocalizationResource = {
...
       },
-      subtitle: undefined,
-      title: undefined,
+      subtitle: 'Encontrando dificuldades? Você pode usar qualquer um destes métodos para acessar.',
+      title: 'Use outro método',
     },
     backupCodeMfa: {
-      subtitle: undefined,
-      title: undefined,
+      subtitle: 'Seu código de recuperação é aquele que recebeu quando configurou a autenticação de dois fatores.',
```
Comment (LauraBeatris) [inline packages/localizations/src/pt-BR.ts:35]:
```suggestion
      subtitle: 'Seu código de recuperação é aquele que recebeu quando configurou a autenticação de dois fatores',
```

File: packages/localizations/src/pt-BR.ts [hunk 6]
```diff
@@ -16,57 +16,57 @@ export const ptBR: LocalizationResource = {
...
-      actionText: undefined,
-      blockButton__backupCode: undefined,
-      blockButton__emailCode: undefined,
-      blockButton__password: undefined,
-      blockButton__phoneCode: undefined,
-      blockButton__totp: undefined,
+      actionLink: 'Precisa de ajuda?',
+      actionText: 'Não possui nenhum destes?',
+      blockButton__backupCode: 'Use um código de recuperação',
+      blockButton__emailCode: 'Enviar código por e-mail para {{identifier}} ',
```
Comment (LauraBeatris) [inline packages/localizations/src/pt-BR.ts:22]:
```suggestion
      blockButton__emailCode: 'Enviar código por e-mail para {{identifier}}',
```

File: packages/localizations/src/pt-BR.ts [hunk 7]
```diff
@@ -16,57 +16,57 @@ export const ptBR: LocalizationResource = {
...
+      blockButton__password: 'Continuar com sua senha',
+      blockButton__phoneCode: 'Enviar código por SMS para {{identifier}}',
+      blockButton__totp: 'Use seu aplicativo de autenticação',
       getHelp: {
-        blockButton__emailSupport: undefined,
-        content: undefined,
-        title: undefined,
+        blockButton__emailSupport: 'Suporte por e-mail',
+        content: 'Se você tiver problemas para acessar sua conta, envie um e-mail para nós e nós o ajudaremos a recuperar o acesso o mais rápido possível.',
```
Comment (LauraBeatris) [inline packages/localizations/src/pt-BR.ts:28]:
```suggestion
        content: 'Se você tiver problemas para acessar sua conta, envie um e-mail para nós e ajudaremos a recuperar o acesso o mais rápido possível.',
```

**Review by LauraBeatris — APPROVED** [review]

**Comment by LauraBeatris** [comment]
Hey @GustavoOS! CI is failing here due to some permission issues with forked repositories. We're working on refactoring those workflows to keep them secure and make it so that it runs for external contributions. 

I'll make sure to keep you updated here once that's in place.

**Comment by GustavoOS** [comment]
any news, @LauraBeatris ?

**Pull request closed without merging**